<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpForm</class>
 <widget class="QMainWindow" name="HelpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_quit"/>
   <addaction name="separator"/>
   <addaction name="action_home"/>
   <addaction name="action_up"/>
   <addaction name="action_back"/>
   <addaction name="action_forward"/>
   <addaction name="action_refresh"/>
  </widget>
  <action name="action_back">
   <property name="icon">
    <iconset resource="resouce.qrc">
     <normaloff>:/image/resource/1x.png</normaloff>:/image/resource/1x.png</iconset>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
   <property name="toolTip">
    <string>后退</string>
   </property>
  </action>
  <action name="action_forward">
   <property name="icon">
    <iconset resource="resouce.qrc">
     <normaloff>:/image/resource/1w.png</normaloff>:/image/resource/1w.png</iconset>
   </property>
   <property name="text">
    <string>前进</string>
   </property>
   <property name="toolTip">
    <string>前进</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="resouce.qrc">
     <normaloff>:/image/resource/2p.png</normaloff>:/image/resource/2p.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="action_home">
   <property name="icon">
    <iconset resource="resouce.qrc">
     <normaloff>:/image/resource/2k.png</normaloff>:/image/resource/2k.png</iconset>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
   <property name="toolTip">
    <string>首页</string>
   </property>
  </action>
  <action name="action_up">
   <property name="icon">
    <iconset resource="resouce.qrc">
     <normaloff>:/image/resource/1s.png</normaloff>:/image/resource/1s.png</iconset>
   </property>
   <property name="text">
    <string>向上</string>
   </property>
   <property name="toolTip">
    <string>向上</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="icon">
    <iconset resource="resouce.qrc">
     <normaloff>:/image/resource/1p.png</normaloff>:/image/resource/1p.png</iconset>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
   <property name="toolTip">
    <string>刷新</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resouce.qrc"/>
 </resources>
 <connections/>
</ui>
